### ISSUE TITLE
[Eclipse-ThirdParty][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for document in "Install dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1367]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. Install dialog will open.
5. In "Install" dialog , tab to document and observe the name property using inspect.

**Actual Result:**
Name property is empty for the document..

**Narrator behaviour:**
Tab
Narrator is not announcing the value. Value information for the document.

**NVDA behaviour:**
edit  read only

**Expected Result:**
Name property should be simple and meaningful.
Name property for the document should be "Available Software".
Narrator should  announce the value. Value for the document .

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![install-available software-name-empty](https://user-images.githubusercontent.com/22956130/35679988-f24cc784-077e-11e8-9668-f637a85ecfc7.jpg)
![install-available software-name-nvda](https://user-images.githubusercontent.com/22956130/35680002-fd3b2168-077e-11e8-9317-f6b89e0abae3.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749823
